artifacts/alfresco_setup.properties

```diff
@@ -0,0 +1,8 @@
+https://alfresco.armcsbs.org,admin,admin
```

Comment:
@ymuwakki could we leverage ConfigMap in kubernetes for this properties file. So all these values inside property file could be given deploy stage.

Reply:
@ymuwakki I thought you said that you are now using environment variables, so shouldn't this file go away?

Reply:
yes. removed.